src/lwc/utilIllustration/utilIllustration.html

```diff
@@ -1045,6 +1045,10 @@
             </svg>
         </template>
         <div class="slds-text-longform">
+            <h2 if:true={hasHeading2}
```

Comment:
@kenlewis-sfdo , @vmartinov-sforg : RD2 forms are using the `util-illustration` and when the page has `h1` but does not have `h2`, an accessibility Jest test fails. Can you please let me know if there is any issue with adding `h2` into the illustration? Other existing forms should continue to work as before.

Reply:
@RadSFDeveloper I don't see any issues with adding `h2` to the `util-illustration`.  Thank you.

Reply:
How does the `heading2` field relate to the `title` field?  Does the title render as an h1?  I wonder if heading2 could be named "subtitle" or something that indicates its usage in relation to the title field.

Reply:
`title` is `h3`.

Reply:
In that case could this field just be called `heading` instead of `heading2`?  It could still render as an h2 element to help the Jest test pass, and that name might add some clarity to its intended use and relationship to the title field.

Reply:
No prob, the change from `heading2` to `heading` has been done.

Reply:
👍🏽 Looks great, thank you!